onnxruntime/core/providers/cpu/ml/tree_ensemble_common.h

```diff
@@ -7,6 +7,9 @@
 #include "core/platform/ort_mutex.h"
 #include "core/platform/threadpool.h"
 
+#define TREEENSEMBLE_BATCHSIZE 256
+#define TREEENSEMBLE_MAXSIZE 8589934592
```

Comment:
constexpr?

I am wondering why we are having so much code in the header? Are we sharing it with other .cc files?